data/blocklist.json

```diff
@@ -2,5 +2,5 @@
     "bing-ads",
     "dbt-event-logging",
     "dbt-utils",
-    "facebook-ads"
+    "facebook-ads",
```

Comment:
```suggestion
    "facebook-ads"
```

Reply:
Oh my bad, I don't know how that got in :)